test/e2e/framework/util.go

```diff
@@ -2350,15 +2350,27 @@ func waitListSchedulableNodesOrDie(c clientset.Interface) *api.NodeList {
 	return nodes
 }
 
+// isNodeTainted checks if a node is tainted and thus not schedulable to pods (without a toleration)
```

Comment:
Not sure if you're asking for more than "what is an example of a toleration"... I suspect you are, but here is an example of a toleration anyway:

https://github.com/kubernetes/kops/blob/master/upup/models/cloudup/resources/addons/dns-controller/v1.4.0.yaml#L23